.github/dependabot.yml

```diff
@@ -20,4 +20,22 @@ updates:
     allow:
       - dependency-type: "direct"
     open-pull-requests-limit: 10
-    rebase-strategy: "disabled"
\ No newline at end of file
+    rebase-strategy: "disabled"
+  - package-ecosystem: devcontainers
+    directory: /
+    schedule:
+      interval: weekly
+  - package-ecosystem: docker
+    directories:
+      - /
+      - ubuntu-ppa
+    schedule:
+      interval: weekly
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+  - package-ecosystem: npm
+    directory: docs
+    schedule:
+      interval: weekly
```

Comment:
@reneleonhardt I'd prefer not to manage these ecosystem dependencies with dependabot.